### DISCUSSION TITLE
Does not support Bagnon on WOW Classic

### DISCUSSION THREAD

**Discussion by ericoshina** [SFX-WoW/Masque#199] [Questions] [answered]
Hello

Right now, Masque does not support the bag addon Bagnon on WOW Classic

Would you have a plan to add this on?

Thanks for update this for Classic. My favortie addon ever.

**Comment by StormFX** [comment, marked as answer]
Masque does not provide support for specific add-ons. That's up to the add-on authors or third parties. In the case of Bagnon, support is provided by [Bagnon_Facade](https://github.com/Jaliborc/Bagnon_Facade).
